lib/mix/tasks/search.embed.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Mix.Tasks.Search.Embed do
+  @moduledoc """
+  Usage: mix #{Mix.Task.task_name(__MODULE__)} <MODULE>
+
+  Embeds the unembedded docs using the model registered in the config
+  """
+  @shortdoc "Embeds the unembedded doc fragments"
+
+  use Mix.Task
+
+  @requirements ["app.start"]
+
+  defp callback({total, done}) do
+    ProgressBar.render(done, total)
+  end
+
+  @impl Mix.Task
+  def run([module_key]) do
+    Search.Embeddings.embed(String.to_existing_atom(module_key), &callback/1)
```

Comment:
```suggestion
  def run([model_name]) do
    Search.Embeddings.embed(String.to_existing_atom(model_name), &callback/1)
```

Reply:
Also, can do a lookup in the models list doing `Atom.to_string(model_key) == model_name`, and if none is found we error with "expected model name to be one of: ...".